components/project-card/project-card.jsx

```diff
@@ -203,6 +203,10 @@ class ProjectCard extends React.Component {
     );
   }
 
+  renderDescription() {
+    return this.props.description.split(`\n`).map(paragraph => <p>{paragraph}</p>);
```

Comment:
since the content came from javascript, `\n` should be good enough. If you're not sure where it came from, then it's typically `.split(/\r?\n/)` to capture the possible carriage return.